src/components/Cards/Cards.jsx

```diff
@@ -4,15 +4,29 @@ import { Row, Col, Badge } from "reactstrap";
 import { GatsbyImage } from "gatsby-plugin-image";
 
 const Cards = ({ title, cover, tags, excerpt, path }) => {
+  let totalTags = [];
+  if (tags.length >= 1) {
+    for (let tag = 0; tag <= 1; tag++) {
+      totalTags.push(tags[tag]);
+    }
+  }
+  let tagCount = tags.length - 2;
```

Comment:
why do you need to do this first. just use tags as is. 

`{tags.map((tag) => <Badge className="mr-2">{tag}</Badge>}`
`{(tags.length -2) > 0 ? "+" : ""}`

Reply:
The if statement check for tags length and  for loop inside it collect first two tags in array
On return we just display the array in which we collected first two's tags and then  , we also check for if our array is of length 2 or less, if it is less then we do not display anything  else if it is greater then 2, we subtract 2 from it and display + "value we get after subtract"

So basically ,
 Tag1 Tag2 +1 more

Conclusion:
We some how have to display first two tags 

The above code you suggested will display all the tags and the  "+ more " , we don't want all tags

Reply:
Did some searching and got solution for removing this  if statement and loop , we can use aarry.slice(0,2) method in JS